Sprint-3/reading-list/script.js

```diff
@@ -20,4 +20,29 @@ const books = [
     bookCoverImage: "https://blackwells.co.uk/jacket/l/9780135957059.jpg",
   },
 ];
+function renderBooks(bookList) {
+  const readingList = document.getElementById("reading-list");
+
+  bookList.forEach((book) => {
+    const listItem = document.createElement("li");
+    listItem.style.backgroundColor = book.alreadyRead ? "green" : "red"; // Set background color directly
```

Comment:
this is a very advanced way to do it, well done!

Reply:
Thank you so much Eliza for reviewing my PR and your kind feedback. Sure, I will do delete the bullet points.